stripe/src/main/java/com/stripe/android/net/PollingNetworkHandler.java

```diff
@@ -123,8 +126,10 @@ public void handleMessage(Message msg) {
 
                 switch (msg.what) {
                     case SUCCESS:
+                        terminated = true;
```

Comment:
Omitting this line was the cause of the "still expires after succeeding bug". Adding the calls to remove callbacks below as a matter of code cleanliness.